apps/chatbot/src/modules/models.py

```diff
@@ -22,7 +22,7 @@
 assert PROVIDER in ["aws", "google"]
 
 
-GOOGLE_PARAM_NAME = os.getenv("CHB_GOOGLE_API_KEY")
+GOOGLE_PARAM_NAME = os.getenv("GOOGLE_PARAM_NAME")
```

Comment:
The environment variable has been previously named CHB_GOOGLE_API_KEY to be compliant with the standard we used in the others
```suggestion
GOOGLE_PARAM_NAME = os.getenv("CHB_GOOGLE_API_KEY")
```

Reply:
ok. Anyway, we need a better way to share better the env variables

Reply:
we have the [.env.example](https://github.com/pagopa/developer-portal/blob/main/apps/chatbot/.env.example) file